web/src/components/storage/device-utils.jsx

```diff
@@ -229,11 +229,11 @@ const DeviceItem = ({ device }) => {
  * @param {Object} props
  * @param {StorageDevice[]} props.devices - Devices to show.
  */
-const DeviceList = ({ devices }) => {
+const DeviceList = ({ devices, ...props }) => {
```

Comment:
It is a bit weird to pass list props to each item. Anyway, we need to rethink selectors and also these lists. I would add a note.